src/app/projects/components/project-settings/components/embeddings/embeddings.component.ts

```diff
@@ -65,18 +86,43 @@ export class EmbeddingsComponent implements OnInit, OnDestroy, OnChanges {
       }));
   }
 
+  prepareEmbeddingPlatforms() {
+    if (this.organization.gdprCompliant && this.isManaged) {
```

Comment:
should only be organization.gdprCompliant since the managed state shouldn't effect the result (organizations on os should have the default correctly set)